src/test/resources/regressions/features/fractional_permissions/fields/fail3.gobra

```diff
@@ -9,10 +9,10 @@ type cell struct{
 
 const divisor = 0
 
-//:: ExpectedOutput(type_error)
+//:: ExpectedOutput(contract_not_well_formed)
 requires acc(&x.val, 1/0) && x.val == 42
```

Comment:
ah here's a division by zero. Is this well-formedness error coming from Silicon? I don't understand why it now behaves differently...

Reply:
Yes, this error comes from silicon. I believe that the reason why it did not work before is that the error was not back-translated to an error at the Gobra level.